beta-pipelines/broad/somatic/single_sample/wgs/gdc_genome/GDCWholeGenomeSomaticSingleSample.wdl

```diff
@@ -141,12 +145,14 @@ task emit_pe_records {
         Array[File]+ fastq2_files
         Array[Object]+ readgroups
         String fastq1_suffix = "_1.fq.gz"
-        Int mem = 2
-        Int disk_space = 10
         Int preemptible = 10
         Int max_retries = 0
         Int cpu = 1
+        Int additioinal_memory_mb = 0
```

Comment:
same misspelling.

Reply:
This as a find and replace, so I twill be everywhere :( fixing